src/components/app/index.tsx

```diff
@@ -38,21 +43,32 @@ const Column: AnyStyledComponent = styled.div`
 `;
 
 const TutorialApp: React.FC = (): JSX.Element => {
+  const [tutorialStep, setTutorialStep] = useGlobal("tutorialStep");
+
   return (
-    <TutorialPage>
+    <StyledTutorialPage>
       <Navbar />
       <MainWrapper>
         <Column>
           <TutorialDisplay />
+          <TutorialSelector />
         </Column>
         <Column>
           <TextEditor />
         </Column>
         <Column>
           <h3 className="text-center">Output will go here</h3>
+          <button
```

Comment:
This was actually my code to see if global state was working. If you don't mind removing this button component I'd really appreciate it.